# Planner configs including perception thresholds, toggles, etc.
system_config:
  ros__parameters:
    max_speed: 50
    acceleration: 0.2
    max_steering: 10
    steering_accel: 0.05